packages/cli/src/commands/install-cluster-command.ts

```diff
@@ -0,0 +1,245 @@
+            "If you do not have this file structure please ensure you've completed the initial setup steps here:\n" +
+            BOOTSTRAP_GUIDE_URL
+        )
+      );
+      printHelpInformation(this.context);
+      return 1;
+    }
+
+    // Read and parse the environment.yaml file using YAML package
+    const environmentYamlContent = await Bun.file(environmentYamlPath).text();
+    const environmentConfig: unknown = YAML.parse(environmentYamlContent);
+
+    let pfStackVersion: string | undefined;
+    if (isEnvironmentConfig(environmentConfig)) {
+      pfStackVersion = environmentConfig.pf_stack_version;
+    }
+
+    if (typeof pfStackVersion !== "string") {
+      this.context.stderr.write(
+        pc.red(
+          "ERROR: pf_stack_version not defined in environment.yaml.\n" +
+            "Please ensure you've completed the initial setup steps in the guide here:\n" +
+            BOOTSTRAP_GUIDE_URL
+        )
+      );
+      printHelpInformation(this.context);
+      return 1;
+    }
+
+    // Warn about SLA target being difficult to change
+    this.context.stdout.write(
+      "\n\u26A0\uFE0F WARNING: SLA target affects your network architecture and is not easily changed later.\n"
+    );
+    this.context.stdout.write(
+      "This determines how many availability zones your infrastructure will span.\n"
+    );
+
+    const proceed = await confirm({
+      message: "Do you want to proceed with the installation?",
+      default: true,
+    });
+
+    if (proceed === false) {
+      this.context.stdout.write("Installation cancelled.\n");
+      return 0;
+    }
+
+    // Prompt for VPC name
+    const vpcName = await input({
+      message: "Enter a name for your VPC:",
+      default: `panfactum-${environment}`,
+    });
+
+    // Prompt for VPC description
+    const vpcDescription = await input({
+      message: "Enter a description for your VPC:",
+      default: `Main Panfactum VPC for the ${environment} environment`,
+    });
+
+    const answers = { pfStackVersion, slaTarget, vpcName, vpcDescription };
+
+    // Write configuration to temp file
+    const configPath = currentDirectory + "/.tmp-panfactum-install-config.json";
+    await Bun.write(configPath, JSON.stringify(answers, null, 2));
+
+    this.context.stdout.write("Starting AWS networking installation...\n");
+
+    // Check if sla_target exists in the environment.yaml file and handle accordingly
+    try {
+      const parsedEnvironmentConfig: Record<string, unknown> = YAML.parse(
+        environmentYamlContent
+      );
+
+      if ("sla_target" in parsedEnvironmentConfig) {
+        // If sla_target exists but is different from the chosen value
+        if (parsedEnvironmentConfig["sla_target"] !== slaTarget) {
+          this.context.stderr.write(
+            pc.red(
+              `ERROR: The environment.yaml file already has an SLA target of ${String(parsedEnvironmentConfig["sla_target"])}, ` +
```

Comment:
I don't think we should ask for the sla_target if it is already set and thus we shouldn't have a conflict here.